app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/DefaultSelector.kt

```diff
@@ -0,0 +1,97 @@
+package com.babylon.wallet.android.presentation.ui.composables
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.babylon.wallet.android.designsystem.theme.RadixTheme
+import com.babylon.wallet.android.presentation.ui.modifier.throttleClickable
+import kotlinx.collections.immutable.ImmutableList
+
+@Composable
+fun <T : SelectorItem<*>> DefaultSelector(
```

Comment:
tbh I am not sure what exactly does this.
Maybe add a description on top?

Reply:
it's dropdown selector. I used it in two places this is why I added that class to reuse, but in the end I switched seed prhsae length selection to tabs